data/korath/korath missions.txt

```diff
@@ -257,3 +257,337 @@ mission "Small Scale Delivery"
+			`	Your communicator notifies you of a payment of <payment>.`
+	npc
+		system "Seketra"
+		fleet "Small Kor Mereti"
+		government "Kor Mereti"
+		to spawn
+			random < 30
+		personality
+			staying
+
+
+mission "Efreti: Distress Call 3"
+	name "Recover the fallen crew of the <EDC-ship>"
+	description "Recover the bodies of the dead <EDC-ship> crew on <destination> and return them to <origin>."
+	source "Laki Nemparu"
+	destination "Seleptra Nak"
+	cargo "Efreti coffins" 13
+	landing
+
+	to offer
+		has "Efreti: Distress Call 2: done"
+
+	on offer
+		conversation
+			`You are relaxing at a spaceport restaurant now that the evacuation is complete when a delegation of Korath approach you somewhat hesitantly. They are dressed in what appears to be much finer clothing than most of the Korath you have met, certainly nothing you would wear to do work in a spaceport. As they make the traditional gesture of greeting, which you return, you are curious both as to what they want and how they will ask.`
+			`	One of them takes a small case out of a pocket and removes a black device that looks like three triangles hooked together. He hisses at it for a moment and then it rises out of his hand. The air around it begins to shimmer and soon a life size hologram of a Quarg appears.`
+			`	"Salutations, far traveler," the Quarg rumbles. "These Korath-Friends are the princes of the clan-company that you have assisted. They thank you for your aid in saving their children and grandchildren. While most grateful they are for the living, they cannot fully mourn the dead who lie on <planet>. They ask me to ask that you travel there and bring their lost ones home to <planet>."`
+			choice
+				`	"I will bring them home."`
+					goto home
+				`	"I will if they pay me."`
+					goto pay
+				`	"I won't risk my ship for the dead."`
+			`	You see the leader of the group visibly sag as the Quarg translates your words. Two younger Korath bear him up as the group turns and leaves.`
+				decline
+
+			label home
+			`	You hadn't realized how tense the group was until you saw them relax as your words reach them. They seem relieved at your answer.`
+				goto title
+
+			label pay
+			`	A ripple of what appears to be dissatisfaction passes through the group, but their leader silences them with a gesture.`
+
+			label title
+			`	The leader turns and speaks to you directly. His eyes never waver from yours even as you wait for the translation. "For many ship skills, parent and child work to learn. For some families, three generations are lost. We will not try to salvage the body box of our kin, for none can bear to fly it. We offer you title to salvage the <EDC-ship> and carry our grief far from here."`
```

Comment:
From the previous PR as well - this might be a little _too_ rough on Quarg translation. I think the player should still be able to understand what the Korath are offering (permission to take the ship). This line may also need to change pending the conversation around whether the ship will be rewarded to the player.

Reply:
Actually, I did improve it a bit. However, I just improved it some more. Do you like it?

Reply:
I rewrote that paragraph again.

There are internal Korath developer discussions going on about how to handle tenses when Korath are giving instructions. Korath languages don't have past or future tenses like we do, mainly for biological reasons. (This would apply to all Korath, including the Efreti.) When they give instructions, they need to express a progression of events and causality. We don't have a final answer yet as to how that should be presented. This is waiting on ponderings from both Ravenshining and I, and future discussions on Swizzle6.

This is a Quarg translation of a Efreti, which is a bit of an odd situation. It may be acceptable for the Quarg to add in tenses, by providing its interpretation of the information. However, because it is a Quarg, I modified the text to sound archaic.

In other words, you're getting an archaic Quarg translation of an Efreti's tense-less instructions.